### PR TITLE
Fixes SR poster errors [Manual Mirror]

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
~~So
TG has random posters chosen by choosing a random poster, then applying variables from that to the random-preset poster.
Uh
I'm not good enough to fix it on their side, and it would require a lot of infrastructure and mapping changes... but our side is a one line change.
So here's this~~

Funny. Mirror bot didn't bring it through.
Good thing the pr was already made lmao
**Manual mirror of https://github.com/tgstation/tgstation/pull/72836**

## How This Contributes To The Skyrat Roleplay Experience
Broken icon bad!

## Changelog

:cl:
fix: Randomized round-start posters now pull from the selected poster's icon file, meaning TG Downstreams can have them in a seperate .dmi (Fixes Skyrat poster icons)
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
> I'm not good enough to fix it on their side, and it would require a lot of infrastructure and mapping changes... but our side is a one line change.

I feel like your fix would work just as well upstream, honestly. I don't quite see how they'd be against it, personally.

**Comment by ghost** [comment]
> > I'm not good enough to fix it on their side, and it would require a lot of infrastructure and mapping changes... but our side is a one line change.
> 
> I feel like your fix would work just as well upstream, honestly. I don't quite see how they'd be against it, personally.

True, I... COULD put the one-line change up there. I'll push it and see how they react, if they want me to fix the whole infrastructure (making posters into mapping helpers that actually place the random one, instead of a poster that takes a random poster's stats) then I'll open this one back up

(https://github.com/tgstation/tgstation/pull/72836)

**Comment by ghost** [comment]
hahaha funny thing, the mirror didn't come in
So i'll just reopen this and remove the edit comment

File: code/game/objects/effects/poster.dm [hunk 1]
```diff
@@ -158,6 +158,7 @@
 	name = initial(selected.name)
 	desc = initial(selected.desc)
 	icon_state = initial(selected.icon_state)
+	icon = initial(selected.icon) //SKYRAT EDIT - Modular Posters
```
Comment (ghost) [inline code/game/objects/effects/poster.dm]:
```suggestion
	icon = initial(selected.icon)
```
Goodbye, edit comment. TG has this exact line now.

**Review by GoldenAlpharex — APPROVED** [review]
Thank you!

**Pull request merged**